### ISSUE TITLE
How to calculate fps？

### ISSUE THREAD

**Issue by tinaZZer** [microsoft/transductive-vos.pytorch#22]
Hello！In the log.txt file during training, the time on the training set and validation set is displayed. And how is the fps obtained? Is time divided by 25? Thank you very much!
